Specs/Number/Chinese/NumberRangeModel.json

```diff
@@ -0,0 +1,422 @@
+      "Resolution": {
+        "value": {
+          "StartValue": 100,
+          "EndValue": 310
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "这些苹果最多100最少20个",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "最多100最少20",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 20,
+          "EndValue": 100
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "这些苹果大概有20~100个",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "20~100",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 20,
+          "EndValue": 100
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "数字的范围是一千到一千五百",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "一千到一千五百",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 1000,
+          "EndValue": 1500
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "数字在1000以上， 1500以下",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "1000以上， 1500以下",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 1000,
+          "EndValue": 1500
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "数字比四分之一大， 比二分之一小",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "比四分之一大， 比二分之一小",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 0.25,
+          "EndValue": 0.5
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "这个数字大于或者等于三千 九百 六十五 ",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "大于或者等于三千 九百 六十五",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 3965
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "他的年龄比三十大一些",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "比三十大",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 30
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "他的年龄不小于三十岁",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "不小于三十",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 30
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "这一批产品大概有五百多个",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "五百多",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": {
+          "StartValue": 500
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "订购一批数目在 五百以上的产品",
```

Comment:
why need "space"?

Reply:
Only to make sure it also works when there're spaces.